### ISSUE TITLE
meek-lite support

### ISSUE THREAD

**Issue by mbrostami** [guardianproject/orbot-apple#34]
Hello,

Is there any plan to add meek-lite to be used in the custom bridges?

**Comment by tladesignz** [comment]
Not really. Basically, meek aka. domain fronting is on the way out. A lot of hosting providers have already taken measures or have announced to install them in the future against it.

The ones who don't do that (yet), should be handled with care and are needed for special purposes, e.g. bootstrapping Snowflake.

Why are you asking?

**Comment by mbrostami** [comment]
@tladesignz 
I think meek is much harder to be detected and also gives an option to send tor traffic over CDN providers regardless of using domain fronting. It can hide the meek server IP address behind CDN. 
There are some small/medium CDN providers still supporting domain fronting. 
Snowflake at the end is p2p connection and can not be integrated with proxy chain tools.

**Comment by tladesignz** [comment]
Sorry for not continuing this conversation. We brought back Meek in the meantime. However, we had to completely remove bridge support from the iOS version, because Tor uses too much RAM for Apple's stupid 50 MB limit on iOS network extensions.

Anyway, closing this now.
